addons/website_forum/models/forum_forum.py

```diff
@@ -138,11 +138,15 @@ def _compute_teaser(self):
 
     @api.depends('post_ids')
     def _compute_last_post_id(self):
-        for forum in self:
-            forum.last_post_id = forum.post_ids.search(
-                [('forum_id', '=', forum.id), ('parent_id', '=', False), ('state', '=', 'active')],
-                order='create_date desc', limit=1,
-            )
+        self.last_post_id = False
+        forums_with_posts = self.filtered('post_ids')
```

Comment:
You could skip the _read_group if no forums_with_posts :) (probably gain a query when creating a forum, such folie)

Reply:
Note that I resisted the walrus urge in this update :)

Reply:
Stay in the light !

Reply:
As you access 'post_ids', you will anyway have to access the m2m ... isn't it counter intuitive ?

Reply:
The method is triggered by a change in `post_ids`, so there should would be no additional request here, no?

Reply:
Anyway it should prefetch the ids, not their actual content, so better avoid prefetching all posts just to fetch the last one, hence keeping the read_group.